### ISSUE TITLE
peer dependencies warning on Angular 5+

### ISSUE THREAD

**Issue by artipixel** [dbfannin/ngx-logger#39]
Getting those warnings when performing 'npm install'

```
npm WARN ngx-logger@1.1.2 requires a peer of @angular/core@^4.3.0 but none is installed. You must install peer dependencies yourself.
npm WARN ngx-logger@1.1.2 requires a peer of @angular/common@^4.3.0 but none is installed. You must install peer dependencies yourself.

```

in package.json: 
`    "@angular/common": "5.2.1",
    "@angular/core": "5.2.1",`

**Comment by davidkarlsen** [comment]
Duplicate of #33

**Comment by dbfannin** [comment]
@davidkarlsen, this has been merged and released in version 2.0.0
